docs/antrea-network-policy.md

```diff
@@ -1061,14 +980,23 @@ spec:
       matchLabels:
         app: client
   egress:
-  - action: Drop
+  - action: Allow
     to:
       - fqdn: "*foobar.com"
+    ports:
+      - protocol: TCP
+        port: 8080
+  - action: Drop      # Drop all other egress traffic, in-cluster or out-of-cluster 
 ```
 
-The above example drops all traffic destined to any FQDN that matches the wildcard
-expression `*foobar.com` originating from any Pod with label `app` set to `client`
-across any Namespace. This feature only works at the L3/L4 level.
+The above example allows all traffic destined to any FQDN that matches the wildcard expression
+`*foobar.com` on port 8080, originating from any Pod with label `app` set to `client`
```

Comment:
I notice that we don't write anywhere that `*` can match multiple subdomains (i.e. it can match dots in the domain name). For example, IIRC `*foobar.com` will match `x.foobar.com`, `x.x.foobar.com`, etc.
We should probably also say that `*` can match 0 or more characters, so `*foobar.com` will also match `foobar.com`

These statements are based on my understanding, so to be verified.

Reply:
Added clarification